src/Raven.Client/Documents/Operations/SqlMigrationOperation.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations
+{
+    public class SqlMigrationOperation : IOperation<SqlMigrationResult>
```

Comment:
Do we want to have those commands available in Client API?

Reply:
I don't think so, no